src/richText.js

```diff
@@ -103,7 +102,18 @@ define([
         }
     });
 
-    CKEDITOR.config.allowedContent = true;
+    CKEDITOR.config.allowedContent = {
+        span: {
+            classes: 'label,label-*',
+            attributes: 'data-output-value,data-value,data-date-format',
+        },
+        i: {
+            classes: 'fcc,fcc-*,fa,fa-*',
+        },
+        output: {
+            attributes: 'value',
+        },
```

Comment:
Before CKEditor we did not do anything special to markup typed in labels (which can be handy when rendering as markdown). Does this have an impact on that?